app/models/concerns/permalinkable.rb

```diff
@@ -10,6 +10,12 @@ module Permalinkable
   module ClassMethods
     def find_by_param!(param)
       find_by_permalink!(param)
+    rescue ActiveRecord::RecordNotFound
```

Comment:
we can just call find_by_id for models that have a id-permalink otherwise you might up in weird places where a stage is named just 1 ... or I guess disallow all number permalinks ...